### PR TITLE
Fix for prediction intervals with pipelines

### LINKED ISSUE #2337: [BUG] Time Series Plotting fixes for models without prediction intervals.
**Describe the bug**
Plotting a model forecast that does not provide prediction intervals still adds an empty prediction interval legend.

**To Reproduce**

```python
model = exp.create_model("ets")
exp.plot_model(model)
```

image

**Expected behavior**
Do not show prediction interval legend if model does not support it.

**Versions**
Current `time_series` branch on repo as of 3/28

### PR DESCRIPTION
**Pull request by ngupta23**
## Related Issuse or bug

Closes #2337 

#### Describe the changes you've made

sktime 0.11.0 and below do not support prediction intervals in pipelines. This PR is a fix for this when the pipeline is empty. In this case, the underlying model is used directly for predictions so that we can get the intervals.

This is a temporary fix until the pipelines can support prediction intervals. Once that is done, we can remove this.

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Code style update (formatting, local variables)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

Unit tests to be run on GH

## Checklist:

- [x] My code follows the style guidelines of this project.
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] I have made corresponding changes to the documentation.
- [ ] My changes generate no new warnings.
- [ ] I have added tests that prove my fix is effective or that my feature works.
- [x] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

### CODE REVIEW

**Comment by ngupta23** [comment]
@Yard1 FYI... Failing test is related to `eda()` (unrelated to time series.

File: pycaret/utils/time_series/forecasting/pipeline.py [hunk 1]
```diff
@@ -58,3 +58,29 @@ def _add_model_to_pipeline(
     pipeline_with_model.steps_[-1][1].steps_.extend([("model", model)])
 
     return pipeline_with_model
+
+
+def _pipeline_tansformations_empty(pipeline: PyCaretForecastingPipeline) -> bool:
```
Comment (Yard1) [inline pycaret/utils/time_series/forecasting/pipeline.py:63]:
```suggestion
def _are_pipeline_tansformations_empty(pipeline: PyCaretForecastingPipeline) -> bool:
```

File: pycaret/time_series/forecasting/oop.py [hunk 2]
```diff
@@ -1003,7 +1005,7 @@ def _setup_display_container(self) -> "TSForecastingExperiment":
         self.logger.info(f"Setup Display Container: {self.display_container[0]}")
         if self.verbose:
             pd.set_option("display.max_rows", 100)
-            print(self.display_container[0].style.apply(highlight_setup))
+            display(self.display_container[0].style.apply(highlight_setup))
```
Comment (Yard1) [inline pycaret/time_series/forecasting/oop.py:1008]: Can we use our Display class here?

**Review by Yard1 — CHANGES_REQUESTED** [review]

File: pycaret/time_series/forecasting/oop.py (hunk 2 above)
Comment (ngupta23) [reply pycaret/time_series/forecasting/oop.py:1008]: I am reusing the one from the regression oop file. Is that not the one to use?

**Review by Yard1 — APPROVED** [review]

**Pull request merged**